packages/comet-extras/src/components/toast/toast.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import Toast, { ToastProps } from './toast';
+import Button from '../../../../comet-uswds/src/components/button/button';
+
+const meta: Meta<typeof Toast> = {
+  title: 'Extras/Toast',
+  component: Toast,
+  argTypes: {
+    id: { control: 'text' },
+    message: { control: 'text' },
+    duration: { control: 'number' },
+    type: { control: 'select', required: true },
+    onClose: { action: 'close' },
+    allowClose: { control: 'boolean' },
+    className: { control: false },
+  },
+};
```

Comment:
Since your story requires a bit more code, you'll need to add some additional configurations here to tell Storybook to render all code. Add this to line 18:
```
parameters: {
    docs: {
      source: {
        type: 'code',
      },
    },
  },
```

Reply:
The modal works similarly: https://github.com/MetroStar/comet/blob/e19b3169d601b1aa0afab380a55c450ab37ec754/packages/comet-uswds/src/components/modal/modal.stories.tsx#L12